Revamping for ubuntu (#51)
pyne/install_scripts: README.md

-------------------

These scripts will install PyNE and its dependencies on the system.

> Install script is available for different versions of both Ubuntu and

Reviewer: ```suggestion
An install script is available for different versions of both Ubuntu and
```

> Mint operating systems. The intention of these

Reviewer: Has anyone actually tested this on Mint?

Reviewer: This was introduced by Kansas State - not sure if we need to keep it?

scripts is that PyNE can be ready to use on a clean install of any of

> the supported operating systems. Furthermore, the user should choose either

Reviewer: Should we define "clean"?  This is mostly about python environments, right?  Should we say that if users have customized/configured their python environment beyond the original OS installation, we can't guarantee that the scripts will work?

-------------------

These scripts will install PyNE and its dependencies on the system.

> An install script is available for different versions of both Ubuntu operating systems. 

Reviewer 2: ```suggestion
An install script is available for different versions of Ubuntu operating systems. 
```

The intention of these scripts is that PyNE can be ready to use on a clean install of any of
the supported operating systems: if users have customized/configured their python environment 
beyond the default OS installation, we can't guarantee that the scripts will work.

> Furthermore, the user should choose either to build a stable version of PyNE or the current

Reviewer 2: ```suggestion
Furthermore, the user should choose either to build a stable release version of PyNE or the current
```

develop branch of PyNE by supplying a second argument. 

> Example for installing the most recent stable branch on Ubuntu 18.04:

Reviewer 2: ```suggestion
Example for installing the most recent stable branch on Ubuntu 20.04:
```

    ./ubuntu.sh stable
    

> Example for installing the develop branch on Mint 18.01:

Reviewer 2: ```suggestion
Example for installing the develop branch on Ubuntu:
```

	
	./ubuntu.sh dev

**!!!WARNING!!!:** 

Those scripts are intended to be used on python3 environment then assumes that 

> your system uses python3 as default.

Reviewer 2: ```suggestion
your system uses python3 as default when the `python` executable is run.
```

To test which default Python version your system is using, run in a terminal:
`python --version`.
If it returns `Python 3.x.y`, you should be able to use our install scripts.
If it returns `Python 2.x.y`, our install 
script will not work as is on your system. You will need to update the default python 

> version.

Reviewer 2: Let's link to a resource about update alternatives here

Docker Builds (*.dockerfile)
----------------------------

These Dockerfiles can be used to build a docker image that has PyNE
installed. The dockerfile chosen will build on the version of Ubuntu

> listed (16.04 or 17.04). Furthermore, the user may choose either

Reviewer 2: just realized I need to update this line in PR #60
